chartfx-chart/src/test/java/de/gsi/chart/plugins/EditAxisTests.java

```diff
@@ -1,7 +1,6 @@
 package de.gsi.chart.plugins;
 
-import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Codacy found an issue: [Avoid unused imports such as 'org.junit.jupiter.api.Assertions'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

Reply:
This is a wrong positive pmd bug: pmd/pmd#2016